tests/test_usecase.py

```diff
@@ -131,9 +131,9 @@ def execute_cmds(self, cmds, capture_output=CAPTURE_OUTPUT):
             exc_class, exc, tb = sys.exc_info()
             if sys.version_info.major == 2:
                 # using six to avoid a SyntaxError in Python 3.x
-                six.reraise(FakeSystemExit, exc, tb)
+                six.reraise(FakeSystemExit, *exc.args, exc_traceback=tb)
```

Comment:
I think this line should not be changed. Am I wrong?

Reply:
The code in the current PR is wrong for Python2 as `six.reraise` looks like its arguments have been renamed moving to Python3.   `six.reraise(FakeSystemExit, *exc.args, tb=tb)` works. 

The reason for this change is dependent on the change of `FakeSystemExit`s base class.  Its so the `code` attribute is set to `code=0` and not `code=SystemExit(0,)`.  It could probably be changed to `reraise(FakeSystemExit, exc.args[0], tb)`.

Reply:
Sorry, what we need is `FakeSystemExit, FakeSystemExit(*exc.args), exc_traceback=tb`.